Shuffle around Boost type_traits, which otherwise causes issues with boost 1.62.
See also: https://bugs.gentoo.org/show_bug.cgi?id=595986

--- a/libMems/AbstractMatch.h
+++ b/libMems/AbstractMatch.h
@@ -16,8 +16,7 @@
 #include "libGenome/gnClone.h"
 #include <vector>
 #include <algorithm>
-#include <boost/type_traits/remove_pointer.hpp>
-#include <boost/type_traits/add_pointer.hpp>
+#include <boost/type_traits.hpp>
 #include <boost/dynamic_bitset.hpp>
 #include <libMems/SlotAllocator.h>
 #include <libMems/configuration.h>
